src/util/gcalUtils.ts

```diff
@@ -186,7 +187,15 @@ export function getRenderedEvents(
             calendarName: event.calendarName,
           });
         } else {
-          const eventStart = moment.utc(date);
+          // rrule returns dates in 'UTC' with 0 offset. See https://github.com/jakubroztocil/rrule#important-use-utc-dates
+          const eventStart = moment([
+            date.getUTCFullYear(),
+            date.getUTCMonth(),
+            date.getUTCDate(),
+            date.getUTCHours(),
+            date.getUTCMinutes(),
+            date.getUTCSeconds(),
+          ]);
```

Comment:
This fixed the issues I was having with timezones being off. I left a comment with a link to where the rrule library talks about it.